WordPress/Classes/Services/PostService.m

```diff
@@ -192,7 +192,12 @@ - (void)uploadPost:(AbstractPost *)post
                     if (success) {
                         success(postInContext);
                     }
-                } failure:failure];
+                } failure:^(NSError *error){
+                    // Sergio Estevao (2018-02-27): even if media fails to attach we are answering with success because the post uploaded sucessfull and the only thing that failed was attaching the media to it.
+                    if (success) {
```

Comment:
Are there not situations where we will want to report failure? Can / should we check that it's due to the attachment?